deploy/charts/rook-ceph/values.yaml

```diff
@@ -156,6 +157,14 @@ csi:
         limits:
           memory: 1Gi
           cpu: 500m
+    - name : csi-omap-generator
```

Comment:
Omap generator need to talk to ceph and it's cephcsi specific , use limit and request same as rbd-plugin